pacman/README.md

```diff
@@ -0,0 +1,155 @@
+# Credits
+
+Credits: [UC Berkeley](http://ai.berkeley.edu/project_overview.html)
+         [OpenAI Interface](https://github.com/sohamghosh121/PacmanGym)
+
+# Installation
+
+
+All the instructions below have been tested under Linux and should work for MacOS.
+
+ - See [Anaconda webpage](https://conda.io/docs/user-guide/install/index.html) for instructions to install Anaconda/Miniconda (recommended).
+
+ - Create your own Anaconda environment and activate it.
+```bash
+conda create -n <env_name> python=3.6
+source activate <env_name>
+```
+
+From now, it is assumed that <env_name> is activated. 
+
+ - Install dependencies.
+```bash
+pip install -U gym numpy stopit argparse pynput pillow
+```
+
+ - Install and use pycodestyle to check PEP8 compliance of Python scripts.
+```bash
+pip install -U pycodestyle
+pycodestyle <yourscript>.py
+```
+
+ - Install and use autopep8 to rewrite Python scripts into PEP8 format.
+```bash
+pip install -U autopep8
+autopep8 --in-place --aggressive --aggressive <yourscript>.py
+```
+
+/!\ Check your file again with pycodestyle.
```

Comment:
use `:warning:`

Reply:
Also, instructions regarding the style should rather go in the evaluation section.